celery_prometheus_exporter.py

```diff
@@ -236,8 +238,8 @@ def __init__(self, app, queue_list):
     def measure_queues_length(self):
         for queue in self.queue_list:
             try:
-                length = self.connection.default_channel.queue_declare(queue=queue, passive=True).message_count
-            except (amqp.exceptions.ChannelError,) as e:
+                length = self.redis_client.llen(queue)
```

Comment:
just making sure: should this be `llen` and not `len`?

Reply:
also what about the case where redis_client is not supplied to this class? Do we only use `QueueLengthMonitoringThread` in this situation? =

Reply:
Yeah it should be llen. As far as I know, only main() instantiates `QueueLengthMonitoringThread` so it should be alright

Reply:
good enough for moi